content/courses/native-onchain-development/paging-ordering-filtering-data-frontend.md

```diff
@@ -151,17 +166,26 @@ async function fetchMatchingContactAccounts(
   connection: web3.Connection,
   search: string,
 ): Promise<(web3.AccountInfo<Buffer> | null)[]> {
-  const accounts = await connection.getProgramAccounts(programId, {
-    dataSlice: { offset: 0, length: 0 },
-    filters: [
-      {
-        memcmp: {
-          offset: 13,
-          bytes: bs58.encode(Buffer.from(search)),
-        },
-      },
-    ],
-  });
+  const accounts = (await connection.getProgramAccounts(
+    new PublicKey(MOVIE_REVIEW_PROGRAM_ID),
+    {
+      dataSlice: { offset: 2, length: 18 },
```

Comment:
Why 2 and 18? Use a variable or at least add a comment.

Reply:
done!

Reply:
I can still see `dataSlice: { offset: 2, length: 18 },` above, No variable, no comment.